board/views.py

```diff
@@ -50,13 +76,30 @@ def get(self, req):
         Returns:
             A JSON representation of the posts.
         """
-        board_hash = self.request.GET.get('board')
-        index = self.request.GET.get('index')
-        amount = self.request.GET.get('amount')
+        try:
+            # Ensure all these parameters exist.
+            board_uuid = req.GET.get('board')
+            index = req.GET.get('index')
+            amount = req.GET.get('amount')
+
+            # Validating each param.
+            board_uuid = UUID(board_uuid, version=4)
+            Board.objects.get(uuid=board_uuid)
+            index = int(index)
+            amount = int(amount)
```

Comment:
I feel like this should be a `400 Bad Request` as per RTC7231:
> The 400 (Bad Request) status code indicates that the server cannot or
   will not process the request due to something that is perceived to be
   a client error (e.g., malformed request syntax, invalid request
   message framing, or deceptive request routing).

where index and amount are supposed to be numbers, and therefore a client error that the server refuses to process.

Reply:
oui